scalableminds/webknossos
# Allow viewing vx workflows via link if organization matches

### URL of deployed dev instance (used for testing):
- https://vxunlisted.webknossos.xyz

### Steps to test:
- Run vx workflow, reporting to this instance
- You should be able to see and list the workflow
- Another user in the same orga should see via link but not list the workflow

### Issues:
- fixes #6619 

------
